lighthouse-core/audits/load-fast-enough-for-pwa.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+const Audit = require('./audit');
+const TTIMetric = require('./time-to-interactive');
```

Comment:
gotta rerun parts of it, but the tracing model is a comptued artifact.

i profiled a run and reruning the TTI.audit() probably is like 3ms max, but we have like 6000ms of main thread time total.